contents/handbook/people/team-structure/team-structure.md

```diff
@@ -11,44 +11,42 @@ We've organised the team into small teams that are multi-disciplinary. [You can
 
 - **[Core analytics](core-analytics)**
     - [Eric Duong](/handbook/people/team#eric-duong-software-engineer) (Team lead, Full Stack Engineer)
+    - [Neil Kakkar](/handbook/people/team#neil-kakkar-software-engineer) (Full Stack Engineer)
```

Comment:
I wonder if we should just mention the team names here and link to their team pages to avoid outdated information. I believe we've changed the structure in the past in the team pages but forgotten about this. We also don't get a ton of views on this page anyways (~100/mo)

Reply:
I would agree with this. I think there should be a many to many relationship allowed for teams <-> members. Especially with PMs. We had PMs shared across product teams very frequently at Uber and other places I have worked.

Reply:
I find it annoying trying to figure out which team someone is on if I have to dig into each team page one at a time to do this and find a simple overview like this more helpful.

I hear you on the outdated info - I'd rather than _not_ have them on the 'deeper' individual team pages, so would remove them from there so we're not duplicating. Then each team page can detail what they're working on just not the folks on it.

Reply:
Done